config/initializers/active_admin.rb

```diff
@@ -34,6 +34,7 @@
   #
   # Default:
   config.default_namespace = :manage
+  # config.skip_before_filter = :authenticate_user!
```

Comment:
IMO this is a necessary behaviour. If this method is failing, try to use `skip_before_action` instead.

Also, owrking here, you can remove this line from the monkey patch in this same file below.